pkg/vulnsrc/ubuntu/ubuntu_test.go

```diff
@@ -33,6 +34,7 @@ func TestVulnSrc_Update(t *testing.T) {
 					Key: []string{"advisory-detail", "CVE-2020-1234", "ubuntu 18.04", "xen"},
 					Value: types.Advisory{
 						FixedVersion: "1.2.3",
+						Status:       types.StatusFixed,
```

Comment:
it should be empty, no?
https://github.com/aquasecurity/trivy-db/pull/407#discussion_r1648346360

Reply:
Test fixed.
Status should be empty with logic change to include it only when status is not fixed.